kafka_consumer/datadog_checks/kafka_consumer/kafka_consumer.py

```diff
@@ -1,21 +1,25 @@
 # (C) Datadog, Inc. 2019-present
 # All rights reserved
 # Licensed under Simplified BSD License (see LICENSE)
+import json
+from collections import defaultdict
 from time import time
 
-from datadog_checks.base import AgentCheck
+from datadog_checks.base import AgentCheck, is_affirmative
 from datadog_checks.kafka_consumer.client import KafkaClient
 from datadog_checks.kafka_consumer.config import KafkaConfig
 
+MAX_TIMESTAMPS = 1000
```

Comment:
Could we add this as a config option in the instance? We don't absolutely need to document it but for some support cases it might be useful to tweak this value